### ISSUE TITLE
July 2019 Compounds for Metabolic/Physchem Evaluation

### ISSUE THREAD

**Issue by edwintse** [OpenSourceMalaria/Series4#72]
The following 3 compounds in the green box will be sent this week to Sue Charman's lab at Monash University for evaluation of their metabolic and physical chemical properties, namely human/mouse liver microsome and rat hepatocyte clearance/half-life, kinetic solubility, logD (pH 7.4). The OHOH compound will be sent from Sydney, while the remaining compounds will be sent from London.

These compounds were chosen for the following reasons:
**295** is the lead OHOH compound so we need the remaining sol/metabolic data for it.
**698** is the most general S4 compound that will act as a baseline (previous compounds didn't possess the 4-OCHF2 group).
**642** is a BCP replacement for phenyl which is supposed to aid with deplanarise and improve solubility.

We will also send an undecided control/repeat compound. Current choices are **155** and **896** (orange box) as these are the most closely related to the key OHOH compound. Are there any other suggestions for a good control compound that we could include other than these two?

![Monash July 2019](https://user-images.githubusercontent.com/18062981/61372939-55d09200-a890-11e9-8661-ea1f506798ea.jpeg)

[Monash 2019 Chemdraw.zip](https://github.com/OpenSourceMalaria/Series4/files/3401694/Monash.2019.Chemdraw.zip)

**Comment by MedChemProf** [comment]
@edwintse I suggest that MMV639565 (OSM-S-272) also be sent as a standard. With the variability of some of the assay results we have seen in the past, I think that at least one compound without the -OCHF2 present is a good idea. MMV639565 also already has a lot of historical PK and stability information associated with it to compare with any newly generated information.

**Comment by mattodd** [comment]
I think MMV693155 has been measured twice for metabolic parameters now, is that right? Meaning it's a pretty good candidate as a control? We have permission to send 4 compounds only... Very nicely the Monash team will also be measuring rat liver microsomal clearance since we've seen remarkably different values for human and mouse, and rat is a better model given that the in vivo model is rat.

**Comment by edwintse** [comment]
Yes, both MMV639565 and MMV693155 have been measured twice in 2017 and 2018. I have some of 155 here in London but I don't have any of 565. If we go with 565, there should be some left in Sydney that @maratsydney can send with the OHOH.

**Comment by MedChemProf** [comment]
@edwintse @mattodd OK. If it is matter that we can only send a limited number, then my suggestion makes less sense. I do believe we have some MMV639565 if needed.  I would just need to know the details of where to send.

**Comment by mattodd** [comment]
Thanks for this, @MedChemProf . I think so long as we have one standard compound, we take care of the consistency question, and we have some of MMV693155 sitting here, so @edwintse can send it tomorrow. Should you send compounds in future, though, I would definitely agree with you sending MMV639565 each time, as you've wisely suggested before in #8 .

OK, so let's keep this Issue open until the results come in.

**Comment by edwintse** [comment]
MMV897698, MMV1794642 and MMV693155 have just been shipped to Monash. The OHOH compound will be shipped on Monday from Sydney.

**Comment by edwintse** [comment]
UPDATE:
The results have come back and are summarised below. Measurements are for human, mouse and rat liver microsomes. The potency for the OHOH needs to be confirmed (will be sent shortly). The potency for the BCP is withheld for the competition.

![RESULTS Monash 2019 Chemdraw](https://user-images.githubusercontent.com/18062981/62863056-a1dfec80-bcff-11e9-9890-76ceafaf132a.png)

**Observations:**

1. LogD values are slightly higher than cLogP for the OHOH and BCP.
2. The OHOH shows excellent solubility, as to be expected with two OH groups. Clearance rates and half-lives are variable across the different microsomes but much better than MMV897698.
3. The BCP performs very well. It has similar solubility to MMV897698, but shows significantly better clearance rates and half-lives.

Data will be entered into the Master List and the image posted to the [temporary biology page](https://github.com/OpenSourceMalaria/Series4/wiki/Biological-Data-currently-not-incorporated-into-the-main-Wiki-sections) on the wiki.

[RESULTS Monash 2019 Chemdraw.zip](https://github.com/OpenSourceMalaria/Series4/files/3492139/RESULTS.Monash.2019.Chemdraw.zip)
